### DISCUSSION TITLE
怎样coremark跑出来的log评估MCU性能

### DISCUSSION THREAD

**Discussion by Inchips-zb** [ingchips/ING918XX_SDK_SOURCE#143] [Q&A]
![image](https://user-images.githubusercontent.com/63443363/223090211-34d3d075-ebf7-4e8d-8909-0b7cbf279648.png)
如上怎样coremark跑出来的log评估MCU性能

**Comment by zjli-2019** [comment]
代码默认为 918 准备，循环测试比较少。而 916 运行速度更快，导致测试时间不足，所以报错。

可参考文档，将循环次数改大：
![image](https://user-images.githubusercontent.com/51557614/223091288-3b5cf514-61ad-42f9-ae07-32d73c5bc686.png)

**Reply by Inchips-zb** [reply]
已按要求设置正常测完，还有疑问918和916分别跑完测试，如何分析两份结果以得到两款mcu性能差异呢

**Reply by Inchips-zb** [reply]
![image](https://user-images.githubusercontent.com/63443363/223093086-8c2c3017-406b-43e5-8586-57c841b541fc.png)

**Reply by zjli-2019** [reply]
倒数第 2 行就是 CoreMark 的分数。
